tests/hawc/apps/animalv2/test_forms.py

```diff
@@ -0,0 +1,115 @@
+from copy import deepcopy
+
+import pytest
+
+from hawc.apps.animalv2 import forms, models
+from hawc.apps.assessment import models as assessment_models
+
+# TODO - worth adding more tests here? We could test trying to pass in e.g. a string where the form expects an integer...but the form
```

Comment:
change comment to make this a TODO - add more tests

Reply:
done